### PR TITLE
Fix duplicate DB when login (backend)

### LINKED ISSUE #214: Duplicate Records in member table.
**Describe the bug**
When a new user logs in through the frontend webpage, two duplicate records are inserted.
As a result, the user can not log in.

**To Reproduce**
1. Go to http://localhost:3000
2. Click on login
3. login with Google 
4. See the error logs from lpvs container

**Logs from lpvs container**
```
lpvs-lpvs-1     | 2023-09-15 00:56:58.047 ERROR 1 --- [nio-7896-exec-7] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is org.springframework.dao.IncorrectResultSizeDataAccessException: query did not return a unique result: 2; nested exception is javax.persistence.NonUniqueResultException: query did not return a unique result: 2] with root cause
lpvs-lpvs-1     |
lpvs-lpvs-1     | javax.persistence.NonUniqueResultException: query did not return a unique result: 2
lpvs-lpvs-1     |       at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:128) ~[hibernate-core-5.6.15.Final.jar!/:5.6.15.Final]
lpvs-lpvs-1     |       at org.hibernate.query.internal.AbstractProducedQuery.getSingleResult(AbstractProducedQuery.java:1672) ~[hibernate-core-5.6.15.Final.jar!/:5.6.15.Final]
lpvs-lpvs-1     |       at org.hibernate.query.criteria.internal.compile.CriteriaQueryTypeQueryAdapter.getSingleResult(CriteriaQueryTypeQueryAdapter.java:111) ~[hibernate-core-5.6.15.Final.jar!/:5.6.15.Final]
lpvs-lpvs-1     |       at org.springframework.data.jpa.repository.query.JpaQueryExecution$SingleEntityExecution.doExecute(JpaQueryExecution.java:198) ~[spring-data-jpa-2.7.15.jar!/:2.7.15]
lpvs-lpvs-1     |       at org.springframework.data.jpa.repository.query.JpaQueryExecution.execute(JpaQueryExecution.java:90) ~[spring-data-jpa-2.7.15.jar!/:2.7.15]
lpvs-lpvs-1     |       at org.springframework.data.jpa.repository.query.AbstractJpaQuery.doExecute(AbstractJpaQuery.java:156) ~[spring-data-jpa-2.7.15.jar!/:2.7.15]
lpvs-lpvs-1     |       at org.springframework.data.jpa.repository.query.AbstractJpaQuery.execute(AbstractJpaQuery.java:144) ~[spring-data-jpa-2.7.15.jar!/:2.7.15]
lpvs-lpvs-1     |       at org.springframework.data.repository.core.support.RepositoryMethodInvoker.doInvoke(RepositoryMethodInvoker.java:137) ~[spring-data-commons-2.7.15.jar!/:2.7.15]
lpvs-lpvs-1     |       at org.springframework.data.repository.core.support.RepositoryMethodInvoker.invoke(RepositoryMethodInvoker.java:121) ~[spring-data-commons-2.7.15.jar!/:2.7.15]
lpvs-lpvs-1     |       at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.doInvoke(QueryExecutorMethodInterceptor.java:160) ~[spring-data-commons-2.7.15.jar!/:2.7.15]
lpvs-lpvs-1     |       at org.springframework.data.repository.core.support.QueryExecutorMethodInterceptor.invoke(QueryExecutorMethodInterceptor.java:139) ~[spring-data-commons-2.7.15.jar!/:2.7.15]
lpvs-lpvs-1     |       at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at org.springframework.data.projection.DefaultMethodInvokingMethodInterceptor.invoke(DefaultMethodInvokingMethodInterceptor.java:76) ~[spring-data-commons-2.7.15.jar!/:2.7.15]
lpvs-lpvs-1     |       at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:123) ~[spring-tx-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:388) ~[spring-tx-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:119) ~[spring-tx-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:137) ~[spring-tx-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at org.springframework.data.jpa.repository.support.CrudMethodMetadataPostProcessor$CrudMethodMetadataPopulatingMethodInterceptor.invoke(CrudMethodMetadataPostProcessor.java:145) ~[spring-data-jpa-2.7.15.jar!/:2.7.15]
lpvs-lpvs-1     |       at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:241) ~[spring-aop-5.3.29.jar!/:5.3.29]
lpvs-lpvs-1     |       at com.sun.proxy.$Proxy117.findByEmailAndProvider(Unknown Source) ~[na:na]
lpvs-lpvs-1     |       at com.lpvs.service.LPVSLoginCheckService.getMemberFromMemberMap(LPVSLoginCheckService.java:56) ~[classes!/:1.1.0]
lpvs-lpvs-1     |       at com.lpvs.service.LPVSLoginCheckService$$FastClassBySpringCGLIB$$cb08147c.invoke(<generated>) ~[classes!/:1.1.0]
```

**Test environment configuration (please complete the following information):**
* Java: v11
* LPVS Release: 1.1.0

### PR DESCRIPTION
**Pull request by kyudori**
# Description

Fixed login error backend part. **#209**, **#213**
Created PR  again due to incorrect commit.

Fixes #214 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [ ] Test Coverage update

# How Has This Been Tested?
- frontend
```
cd frontend
npm install
npm start
```
- backend
```
mvn clean install
cd target
java -jar lpvs-*.jar
```

**Test Configuration**:
* Java: v11
* LPVS Release: v1.1.0

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by tiokim** [comment]
Please add the `UNIQUE` constraint to the sql file like below.  

```
--- a/src/main/resources/database_dump.sql
+++ b/src/main/resources/database_dump.sql
@@ -168,7 +168,8 @@ CREATE TABLE `member` (
   `name` varchar(255) NOT NULL,
   `nickname` varchar(255) DEFAULT NULL,
   `provider` varchar(10) NOT NULL,
-  `organization` varchar(255) DEFAULT NULL
+  `organization` varchar(255) DEFAULT NULL,
+  UNIQUE KEY `UK7blitxweiw758g32lg3bhavf6` (`email`,`provider`)
```

**Comment by kyudori** [comment]
I made a commit reflecting the your request. 
Please review it. @tiokim
- [Add the UNIQUE constraint.](https://github.com/Samsung/LPVS/pull/215/commits/6b4c7cb77b67d94027df120d62d99ed7b35c1193)

File: src/main/resources/database_dump.sql [hunk 1]
```diff
@@ -169,6 +169,7 @@ CREATE TABLE `member` (
   `nickname` varchar(255) DEFAULT NULL,
   `provider` varchar(10) NOT NULL,
   `organization` varchar(255) DEFAULT NULL
+UNIQUE KEY `unq_member` (`email`,`provider`)
```
Comment (tiokim) [inline src/main/resources/database_dump.sql:172]: You need to add a comma at the end of the 171 line. 
Please be careful about indentation.

```suggestion
  `organization` varchar(255) DEFAULT NULL,
  UNIQUE KEY `unq_member` (`email`,`provider`)
```

**Review by tiokim — APPROVED** [review]
LGTM

**Review by o-kopysov — APPROVED** [review]
LGTM

**Pull request merged**